src/Umbraco.Web/PublishedCache/PublishedCacheBase.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Umbraco.Web.PublishedCache
 {
-    abstract class PublishedCacheBase : IPublishedCache
+    public abstract class PublishedCacheBase : IPublishedCache
```

Comment:
This could get a <summary> saying something along the lines MS use: Internal, not intended for use. (Except mocking)